sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -519,21 +519,15 @@ def start_copy_from_url(self, source_url, **kwargs):
 
         :param str source_url:
             Specifies the URL of the source file.
-        :param file_permission_copy_mode: Specifies the option to copy file
```

Comment:
I think the permission is classified as ACL? So probably we don't want to merge it into SMB Properties?
[https://microsoft.sharepoint.com/:w:/r/teams/AzureStorage/_layouts/15/guestaccess.aspx?e=Aoz2x5&CID=AC082D06-12E2-4488-9A51-E37C83E2451F&wdLOR=c342E3E2B-B2C0-4AAE-B060-695A6C826E91&share=EcFOZrkUnFpNvcKZldali64B-cdzo3TYc_6uMRangP029A](https://microsoft.sharepoint.com/:w:/r/teams/AzureStorage/_layouts/15/guestaccess.aspx?e=Aoz2x5&CID=AC082D06-12E2-4488-9A51-E37C83E2451F&wdLOR=c342E3E2B-B2C0-4AAE-B060-695A6C826E91&share=EcFOZrkUnFpNvcKZldali64B-cdzo3TYc_6uMRangP029A)

Reply:
Yes but its behaviour is inconsistent with all the other properties that simply take a value of "source" rather than have a separate "mode" switch. This makes it confusing for users, so I would like to hide it....

Reply:
Just talked to other SDKs and it seems they are having them as params...

Reply:
All params now flattened, giving a similar experience to `create_file`